Cargo.toml

```diff
@@ -32,6 +32,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 default = ["process-mux"]
 process-mux = []
 native-mux = ["openssh-mux-client"]
+scp = []
```

Comment:
Hmm, why call this `scp`? That's already a separate command, but not one that this implementation uses as far as I can recall?

Reply:
Yeah, but I couldn't figure out an alternative name though.

Reply:
I think I'd go with something more descriptive, like `sh-file-opts`.

Alternatively we _could_ delete the module entirely. After all, There's nothing magical in there, and maybe it's better to shrink the crate's API and have users invoke `cat` and friends if they want to do file ops themselves..

Reply:
Well, if removing this module altogether is acceptable, then I will definitely remove it instead of keeping it around.

Reply:
Yeah, let's do that :+1:

Reply:
Done!